first.rb

```diff
@@ -0,0 +1,136 @@
+students = [
+  {
+      first_name: 'John',
+      last_name: 'Doe'
+  },
+  {
+      first_name: 'Jane',
+      last_name: 'Doe',
+  },
+  {
+      first_name: 'Jennifer',
+      last_name: 'Aniston',
+  }
+]
+
+upper_case_full_names = []
+name1 =[]
+theString=''
+student.each do |key,val|
```

Comment:
Good work! but I think you meant ```students```, not  ```student```, Also in the array, there is not a key but there are index and value